apis/python/tests/test_sparse_nd_array.py

```diff
@@ -257,6 +257,43 @@ def test_sparse_nd_array_read_write_sparse_tensor(
     assert tensors_are_same_value(t, data)
 
 
+@pytest.mark.parametrize(
```

Comment:
Unless I'm missing something, this test is an almost verbatim copy of the previous test.  The same thing could be accomplished by adding the "corner read" test to the above, eg, tacking onto the above test:

```
    # Read back and validate the "corner"
    b = soma.SparseNdArray(tmp_path.as_posix())
    t = next(b.read_sparse_tensor((0,) * len(shape), format=format))
    assert t.shape == shape
```

Rather than duplicating the whole thing.

Apologies if my visual scan missed something else different...